rpcs3/util/simd.hpp

```diff
@@ -597,12 +597,14 @@ inline void gv_zeroupper()
 #if defined(ARCH_X64)
 	if (!g_use_avx)
 		return;
-#if defined(_M_X64)
+#ifdef __AVX__
```

Comment:
Compile-time check for a runtime feature. Same with the __RTM__ check in the other block. Afaik the __AVX__ macro isn't enabled in CI.

Reply:
I will change it to `ifdef _MSC_VER`. To use `_mm256_zeroupper` you need to compile with `-mavx` or use [function multiversioning](https://gcc.gnu.org/onlinedocs/gcc/Function-Multiversioning.html), but this is an inline function so function multiversioning would not work.

Reply:
Our CI builds linux with -msse2 afaik. How was this working in master?

Reply:
`_M_X64` is only defined on Windows and MSVC was the only compiler.